Sming/SmingCore/Network/TcpServer.cpp

```diff
@@ -128,7 +128,7 @@ err_t TcpServer::onAccept(tcp_pcb *clientTcp, err_t err)
 void TcpServer::onClient(TcpClient *client)
 {
 	activeClients++;
-	debugf("TcpServer onClient  %s, activeClients = %d\r\n ",client->getRemoteIp().toString().c_str(),activeClients);
+	debugf("TcpServer onClient  %s\r\n ",client->getRemoteIp().toString().c_str());
```

Comment:
*if* you want to change that the ideal thing to do would be squash the second commit and the first together, but it doesn't look like the original repository still exists so i'd probably just leave it